src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseStatistics.hpp

```diff
@@ -109,8 +109,12 @@ class CompositionalMultiphaseStatistics : public FieldStatisticsBase< Compositio
 
     /// region phase mass
     array1d< real64 > phaseMass;
+    /// immobile region phase mass
+    array1d< real64 > trappedPhaseMass;
```

Comment:
```suggestion
    /// region phase mass (trapped and non-trapped)
    array1d< real64 > phaseMass;
    /// trapped region phase mass
    array1d< real64 > trappedPhaseMass;
```
just to avoid confusion because the "trapped" saturation can be fully mobile given the definition that we have adopted.